connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -1065,12 +1065,20 @@ class SparkConnectPlanner(
       numPartitionsOpt)
   }
 
-  private def transformCollectMetrics(rel: proto.CollectMetrics, planId: Long): LogicalPlan = {
```

Comment:
We don't need to semantically validate `CollectMetrics`? cc @allisonwang-db

Reply:
Updated back to use `planId` here.